rts/Lua/LuaMetalMap.cpp

```diff
@@ -31,6 +31,11 @@ bool LuaMetalMap::PushCtrlEntries(lua_State* L)
 	return true;
 }
 
+/***
+ * @function Spring.GetMetalMapSize
+ * @return integer x X coordinate in worldspace/16.
```

Comment:
There's `Game.metalMapSquareSize` named constant that Lua should use instead of the magic 16.

Reply:
https://github.com/beyond-all-reason/spring/issues/2120